client/src/components/CreateQuestion.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <div>
+    <div class="container">
+      <h2>Create question</h2>
+      <input class="input-textarea" type="text" placeholder="Question" v-model="text" />
+      <ul id="answers">
+        <li v-for="answer in answers" :key="answer.localId">
+          <input type="checkbox" :value="answer.id" v-model="correctAnswers" />
+          <input class="input-textarea" type="text" placeholder="Answer" v-model="answer.text" />
+        </li>
+      </ul>
+      <button class="signup-form-button active-theme" v-on:click.prevent="addNewAnswer">Add Answer</button>
+      <button
+        class="signup-form-button active-theme"
+        v-on:click.prevent="submitQuestion"
+      >Submit question</button>
+      <button class="signup-form-button active-theme" v-on:click.prevent="gotoQuiz">Finish</button>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapActions } from 'vuex'
+export default {
+  name: 'CreateQuestion',
+  data () {
+    return {
+      text: '',
+      answers: [
```

Comment:
so, the number of answers is fixed to 4?

Reply:
No, there is a button that adds more of them; 4 are the initial state

Reply:
2 sounds more reasonable if you cant remove